src/lib/ndpi_main.c

```diff
@@ -2126,6 +2126,10 @@ static void ndpi_init_protocol_defaults(struct ndpi_detection_module_struct *ndp
               "TeslaServices", NDPI_PROTOCOL_CATEGORY_NETWORK,
               ndpi_build_default_ports(ports_a, 0, 0, 0, 0, 0) /* TCP */,
               ndpi_build_default_ports(ports_b, 0, 0, 0, 0, 0) /* UDP */);
+  ndpi_set_proto_defaults(ndpi_str, 1 /* cleartext */, 0 /* nw proto */, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_PTPV2,
+			  "PTPv2", NDPI_PROTOCOL_CATEGORY_SYSTEM_OS,
+			  ndpi_build_default_ports(ports_a, 0, 0, 0, 0, 0) /* TCP */,
+			  ndpi_build_default_ports(ports_b, 0, 0, 0, 0, 0) /* UDP */);
```

Comment:
Could you add 319 and 320 as default ports for UDP, please? They are reserved by IANA

Reply:
Yea, sure.